src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -17,6 +17,26 @@ Author: Kunjian Song, kunjian.song@postgrad.manchester.ac.uk
 
 namespace SolidityGrammar
 {
+std::set<std::string> uintTypeStringSet = {
+  "uint8",   "uint16",  "uint24",  "uint32",  "uint40",  "uint48",  "uint56",
+  "uint64",  "uint72",  "uint80",  "uint88",  "uint96",  "uint104", "uint112",
+  "uint120", "uint128", "uint136", "uint144", "uint152", "uint160", "uint168",
+  "uint176", "uint184", "uint192", "uint200", "uint208", "uint216", "uint224",
+  "uint232", "uint240", "uint248", "uint256",
+};
+std::map<std::string, ElementaryTypeNameT> uintStringToTypeMap = {
```

Comment:
Same as the comment in the header: can't this be `const`? Also, why is this symbol `uintStringToTypeMap` exported in the `SolidityGrammar` namespace? Couldn't it be local to this translation unit by defining it outside the namespace as `static`?